### PR TITLE
L/M -> survey_code

### PR DESCRIPTION
**Pull request by Piwington**

### CODE REVIEW

**Review by KrisRogos — APPROVED** [review]

**Review by kingmushroom — APPROVED** [review]
Looks good

File: strata_period_method.py
Comment (thomashenson) [inline strata_period_method.py:39]: Is this to be changed to the correct name or is it actually called survey_column in the data? As in enrichent it's simply called 'survey'.

File: strata_period_wrangler.py
Comment (thomashenson) [inline strata_period_wrangler.py:69]: Ah, I see. It's being passed in as an env var. Will this simply be the survey code that wants to be run, then?

**Review by thomashenson — COMMENTED** [review]
Just need to confirm a few things before approval.

File: strata_period_wrangler.py
Comment (Piwington) [reply strata_period_wrangler.py:69]: This is the name of the column that the survey code lives in.

**Pull request merged**
